tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -323,8 +326,9 @@ export function makeMatMulPackedSource(
       let globalRowStart = i32(workgroupId.y) * ${tileAOuter};
 
       let numTiles = ${
-      splitK ? '1' : '(uniforms.dimInner - 1) / TileInner + 1'};
-      var kStart = ${splitK ? 'i32(globalId.z) * TileInner' : '0'};
+      splitK ? `${splitedDimInner / tileInner}` :
```

Comment:
Same here

Reply:
Change it to `Math.ceil(splitedDimInner / tileInner)`. Thanks.